dictionary_test.py

```diff
@@ -0,0 +1,14 @@
+import unittest
+
+from dictionary import dictinary
+
+class DictTest(unittest.TestCase):
+
+    def test_dictinary(self):
+        self.assertEqual(dictinary(3,8), {3: 9, 4: 16, 5: 25, 6: 36, 7: 49})
```

Comment:
Write a test that loops through the dictionary and asserts whether each value is the square of each key.
Hint:
```py
for key, value in generated_dictionary:
    # do the assertion here
```

Reply:
Hi @dennisja , I have tried to use a loop to test whether each value is a square of each key.
You may look through the recent changes please